tinker/tinker_controller.py

```diff
@@ -26,11 +26,73 @@
 from flask import abort, current_app, render_template, request, Response, session
 from flask import json as fjson
 from requests.packages.urllib3.exceptions import SNIMissingWarning, InsecurePlatformWarning
+from werkzeug.datastructures import ImmutableMultiDict
 
 # Local
 from tinker import app, cascade_connector, sentry
 
 
+class EncodingDict(object):
+    # This class was created because all the POST methods that take in a form of data return strings as unicode, which
+    # doesn't fully encode to String using the default str() method. Rather than go through the whole project and make
```

Comment:
This is a little misleading. It actually encodes to String just fine. Its just Cascade that doesn't like the form the String is

Reply:
I agree. I'll reword this.